src/data/list/prod_monoid.lean

```diff
@@ -26,6 +26,14 @@ begin
   { rw [list.repeat_succ, list.prod_cons, ih, pow_succ] }
 end
 
+@[to_additive]
+lemma prod_eq_pow_of_forall_eq [monoid α] (l : list α) (m : α) (h : ∀ (x : α), x ∈ l → x = m) :
```

Comment:
```suggestion
lemma prod_eq_of_forall_eq [monoid α] (l : list α) (m : α) (h : ∀ (x : α), x ∈ l → x = m) :
```
To match the `le` version below. Or change that one, since having pow in the name probably makes it easier to find both

Reply:
Will change the `le` lemma